Automattic/newspack-plugin
# feat: (IA) new dashboard schema and app boilerplate

### All Submissions:


### Changes proposed in this Pull Request:

* Adds new server side schema i.e. `wp_localize_script`.
* Adds initial, new, Dashboard React App.
* Adds support for typescript.
* Adds default `undefined` values to Footer and Wizard component props, to appease TypeScript.

Note: where `'href' => '#', // @TODO`'s exist, these are new future pages and will be added as each app is created.

### How to test the changes in this Pull Request:

1. Checkout this branch and build assets i.e. `git checkout feat/ia-dash-initial && npm run build`
2. Navigate to new app - /wp-admin/admin.php?page=newspack#/
3. Observe you see the below:

![Screenshot 2024-03-25 at 21 57 07](https://github.com/Automattic/newspack-plugin/assets/3676975/28e7aa90-48b0-48ea-bf8d-ae86e603d815)

### Other information:

* [] Have you written new tests for your changes, as applicable?
